onnxruntime/onnxruntime-engine/src/test/java/ai/djl/onnxruntime/engine/OrtTest.java

```diff
@@ -73,6 +73,7 @@ public void testOrt() throws TranslateException, ModelException, IOException {
     public void testStringTensor()
             throws MalformedModelException, ModelNotFoundException, IOException,
                     TranslateException {
+        System.setProperty("ai.djl.onnx.disable_alternative", "true");
```

Comment:
How about putting this functionality into Criteria?

Reply:
We cannot simply added to criteria, it is usually scoped into a series of operations